src/components/quit/Quit.test.jsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```

Comment:
Codacy has a fix for the issue: [Replace `Adapter·from·'@wojtekmaj/enzyme-adapter-react-17` with `React·from·'react`](https://app.codacy.com/gh/msaperst/number-munchers/pullRequest?prid=8124485)
```suggestion
import React from 'react';
```